paasta_tools/kubernetes_tools.py

```diff
@@ -1369,14 +1369,18 @@ def list_deployments(
     tail_lines_dict: MutableMapping[str, List[str]] = {"stdout": [], "stderr": []}
     for container in pod.spec.containers:
         if container.name != HACHECK_POD_NAME:
-            tail_lines_dict["stdout"].append(
-                kube_client.core.read_namespaced_pod_log(
-                    name=pod.metadata.name,
-                    namespace="paasta",
-                    container=container.name,
-                    tail_lines=num_tail_lines,
+            try:
+                tail_lines_dict["stdout"].append(
+                    kube_client.core.read_namespaced_pod_log(
+                        name=pod.metadata.name,
+                        namespace="paasta",
+                        container=container.name,
+                        tail_lines=num_tail_lines,
+                    )
                 )
-            )
+            except ApiException as e:
+                log.error(e)
+
```

Comment:
The problem with this is that `paasta status` just won't get anything back, so users won't know the difference between something going wrong and there just not being any stdout/stderr.  Ideally we'd follow the example of the mesos version: https://github.com/Yelp/paasta/blob/v0.93.2/paasta_tools/mesos_tools.py#L414